f32-py38/s2i_assemble.patch

```diff
@@ -1,46 +1,72 @@
-diff --git a/assemble b/assemble
-index c6515b5..3da074c 100755
---- a/assemble
-+++ b/assemble
-@@ -18,20 +18,6 @@ function virtualenv_bin() {
-   fi
+--- a/asseble	2020-06-05 09:22:15.714337470 +0200
++++ b/assemble	2020-06-05 09:21:48.059105881 +0200
+@@ -14,30 +14,8 @@
+     python3.7 -m venv $1
```

Comment:
This is for 3.8 or 3.7? dirname suggests 3.8

Reply:
Nice catch! Adjusted, the build seems to work.